### DISCUSSION TITLE
Clarification from August Update

### DISCUSSION THREAD

**Discussion by JMoodyFWD** [phalcon/phalcon#92] [Q&A] [answered]
Hey All,

I've had my head down building a pretty large project in Phalcon 4, but I just noticed the official post from August, The Future of Phalcon) and I'm wondering a couple of things.

1.) To clarify on Phalcon 5 becoming "Native PHP", this means it will no longer be a framework powered by a PHP extension but rather as a package of files that will have to be loaded within a project?

2.) If "1" is an accurate assumption, is the estimated performance impact going to be negligible?

**Comment by Jeckerson** [comment, marked as answer]
Hello!

1.) yes, `composer require phalcon/phalcon` - you already can do that, but there are missing classes.
2.) nothing to tell specifically atm, some **micro benchmarks** showed 20-30% performance degradation. Any way, stay tuned for more detailed information.

**Reply by JMoodyFWD** [reply]
Thanks for the clarification!
